README.md

```diff
@@ -1 +1,50 @@
-# angular-what-is-angular-readme
+# What is Angular and why does it exist?
+
+## Overview
+
+In this lesson we're going to be looking into AngularJS (commonly referred to as just "Angular"), an open-source framework that was released in 2009. It is currently being mainly maintained by Google as well as a massive community of developers.
+
+It aims to solve the problems faced when creating single page applications, implementing a MVVM pattern to separate presentation and business logic.
+
+## Objectives
+
+- Describe what Angular's role in the web platform is
+- Describe the issues Angular looks to solve
+
+## Frontend Frameworks
+
+Ever wondered how Facebook can have a different timeline for millions of users? How Twitter can display tweets for 300 million people? That's where frontend frameworks come in. 
+
+Frontend frameworks have grown massively in the last 3 years. They help developers structure their applications and create interactive, super quick applications.
+
+### Life before frameworks
+
+Before we had the pleasure of using frontend frameworks, code would be written very procedurally. JavaScript code would become unwieldy as there was no easy way to structure code for each page that the client would see.
+  
+You'd be limited to just simple JavaScript interactions, such as submitting a form via AJAX. Anything more wouldn't have been scalable, resulting in a hard to debug/manage codebase. 
+
+### Frontend frameworks save the day
+
+Imagine being able to write complex JavaScript applications, having a framework deal with scaling your code, splitting it out into logical chunks and not have to worry about one part breaking another.. Sounds like a dream! Oh wait.. Angular does that for us!
+
+## Traditional Rendering
+
+Before mainstream JavaScript usage, the server would do everything for us. When we'd go to a website, everything we needed on that page would be given to us there and then. For instance, if you go to http://wikipedia.org, the server loads the content and sends it back to us.
+
+The problem with this is that data displayed on the page could only be updated when the user refreshes. This also meant no smooth transitions between pages (every page the user navigated to would be reloaded in the web browser).
+
+We'd also then have all of our frontend logic mixed in with the backend logic - resulting in a horrible mess if not structured properly!
+
+## The role of Angular
```

Comment:
This section should focus on things that are unique to Angular. This can be an non-technical explanation. Why is Angular so popular? Why do people choose it over other front-end frameworks?